plugin/shell_plugin.go

```diff
@@ -0,0 +1,13 @@
+package plugin
+
+import (
+	"github.com/alibaba/sealer/utils"
+)
+
+type Sheller struct {
+	role string
+	data string
+}
+func (s Sheller) Run(context Context, phase Phase){
+	s.SSH.CmdAsync()
```

Comment:
You need new ssh first.https://github.com/alibaba/sealer/blob/main/utils/ssh/ssh.go#L55
```
ssh := NewSSHByCluster(context.Cluster)
ssh.CmdAsync....
```